readme.md

```diff
@@ -67,6 +67,17 @@ By defualt we remember the last 50 items in your clipboard. To change this add a
 }
 ~~~
 
+The clipboard monitor update interval is `1` second (except `3` seconds for Linux system) by default, however, user can customize the interval by setting `updateInterval` option in the configuration. The interval has to be no less than `250` ms.
```

Comment:
Sure, I will unify the unit to `ms`.

About the minimal interval, the plugin system's default is `100` ms, however, since the performance issue in the clipboard, I increased the minimum interval to `250` ms.

The reason is it's not only Linux has the performance hit, it's on the macOS as well if the clipboard contains image, image comparison in each round will cost about `1000` ms or more on my laptop if it's a photo, so increasing the minimum interval will let CPU has some rest in the between.